zerver/lib/markdown/api_arguments_table_generator.py

```diff
@@ -163,18 +176,72 @@ def render_table(self, arguments: Sequence[Mapping[str, Any]]) -> List[str]:
             else:
                 deprecated_block = ""
 
-            table.append(
-                argument_template.format(
-                    argument=argument.get("argument") or argument.get("name"),
+            if "object" in data_type:
+                if "schema" in argument:
+                    object_schema = argument["schema"]
+                else:
+                    object_schema = argument["content"]["application/json"]["schema"]
+
+                if "items" in object_schema and "properties" in object_schema["items"]:
+                    object_block = self.render_object_details(object_schema["items"], str(name))
+                elif "properties" in object_schema:
+                    object_block = self.render_object_details(object_schema, str(name))
+                # TODO: There are some endpoint parameters with object properties that are not defined in `zerver/openapi/zulip.yaml`
```

Comment:
This seems like something we can try to eliminate in a follow-up pass; they're probably all oversights.  But I'd be happy merging this as-is, and then doing that as a next mini project; likely adding an assert in that `else` block and then running `test-backend test_docs` (I think that's the one that renders every API endpoint) should catch any entries with this issue.

Reply:
Agreed. I added the comment as a reminder thinking of that being a mini follow-up to this update. I found the error while checking the html diff on the api documentation and have the two endpoints currently without object definitions noted.

So that might be another argument for trying to figure out a way to add an html diff comparison (for both api and help documentation) as a shared tool or test for the development community. The way I have mine set up, it gives me an error log of any link that doesn't return '200' as its status code.